### PR TITLE
feat: [SEI-4003]: Re-Introduce Old navigation for SEI

### PR DESCRIPTION
**Pull request by sharath-levelops**
### Summary

Add SEI old side navigation to harness-core-ui

#### Screenshots
Screenshot 2023-11-08 at 7 20 00 PM
Screenshot 2023-11-08 at 7 20 34 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Review by ajay-levelops — DISMISSED** [review]

File: configs/devServerProxy.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-const baseUrl = process.env.BASE_URL ?? 'https://qa.harness.io/gateway'
+const baseUrl = process.env.BASE_URL ?? 'https://stress.harness.io/gateway'
```
Comment (harshilbits) [inline configs/devServerProxy.config.js:8]: Please revert this change.

File: src/modules/75-sei/RouteDestinations.tsx [hunk 2]
```diff
@@ -40,8 +46,30 @@ const InsightsResourceModalBody = React.lazy(() => import('sei/InsightsResourceM
...
+      <Redirect
+        to={routes.toSEIInsights({
+          accountId,
+          orgIdentifier,
+          projectIdentifier
+        })}
+      />
+    )
+  } else {
+    return <Redirect to={routes.toModuleHome({ accountId, module: 'sei' })} />
```
Comment (harshilbits) [inline src/modules/75-sei/RouteDestinations.tsx:66]: Please use the predefined constants of module sei

File: src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+  const projectTabContent = useMemo(
+    () => (
+      <>
+        <ProjectSelector
+          onSelect={data => {
+            updateAppStore({ selectedProject: data })
+            history.push(
+              routes.toSEIInsights({
+                projectIdentifier: data.identifier,
+                orgIdentifier: data.orgIdentifier || /* istanbul ignore next */ '',
```
Comment (harshilbits) [inline src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx:64]: we are we doing `/* istanbul ignore next */` here ?

File: src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+                  />
+                )}
+                {canViewCollections && (
+                  <SidebarLink
+                    label={getString('common.purpose.sei.collections')}
+                    to={routes.toSEICollection({ accountId, projectIdentifier, orgIdentifier })}
+                  />
+                )}
+                <SidebarLink
+                  to={routes.toAccessControl({ accountId, projectIdentifier, orgIdentifier, module: 'sei' })}
```
Comment (harshilbits) [inline src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx:97]: use the module constants of sei.

File: src/modules/75-sei/RouteDestinations.tsx [hunk 5]
```diff
@@ -40,8 +48,30 @@ const InsightsResourceModalBody = React.lazy(() => import('sei/InsightsResourceM
 // eslint-disable-next-line import/no-unresolved
 const InsightsResourceRenderer = React.lazy(() => import('sei/InsightsResourceRenderer'))
 
+const RedirectToSEIModule = (): React.ReactElement => {
+  // const { accountId } = useParams<ProjectPathProps>()
```
Comment (harshilbits) [inline src/modules/75-sei/RouteDestinations.tsx:52]: remove the commented code.

Comment (sharath-levelops) [reply src/modules/75-sei/RouteDestinations.tsx]: Removed

File: src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx (hunk 4 above)
Comment (sharath-levelops) [reply src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx]: Addressed

File: src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx (hunk 3 above)
Comment (sharath-levelops) [reply src/modules/75-sei/SideNav/NavV1/SEISideNav.tsx]: No. Removed the comment

File: src/modules/75-sei/RouteDestinations.tsx (hunk 2 above)
Comment (sharath-levelops) [reply src/modules/75-sei/RouteDestinations.tsx]: Addressed

File: configs/devServerProxy.config.js (hunk 1 above)
Comment (sharath-levelops) [reply configs/devServerProxy.config.js]: Addressed

**Review by harshilbits — APPROVED** [review]

**Review by ajay-levelops — DISMISSED** [review]

**Review by ajay-levelops — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Review by jenil-harness — APPROVED** [review]
